ui/src/app/store/staticroute/types/actions.ts

```diff
@@ -0,0 +1,17 @@
+import type { StaticRoute } from "./base";
+import type { StaticRouteMeta } from "./enum";
+
+export type CreateParams = {
+  destination: StaticRoute["destination"];
+  gateway_ip: StaticRoute["gateway_ip"];
+  metric: StaticRoute["metric"];
+  source: StaticRoute["source"];
+};
+
+export type UpdateParams = {
```

Comment:
This could also be something like:
```suggestion
export type UpdateParams = {
  [StaticRouteMeta.PK]: StaticRoute[StaticRouteMeta.PK];
} & Partial<CreateParams>;
```